Source/Models/HYPFormTarget.m

```diff
@@ -274,6 +292,7 @@ + (void)filteredTargets:(NSArray*)targets
             case HYPFormTargetActionHide:
                 if (![hidden containsObject:target]) [hidden addObject:target];
                 break;
+            case HYPFormTargetActionClear:
```

Comment:
Does `HYPFormTargetActionUpdate` has the same behavior `HYPFormTargetActionClear`?